Cargo.toml

```diff
@@ -113,7 +113,7 @@ openzeppelin-stylus = { path = "contracts" }
 openzeppelin-stylus-proc = { path = "contracts-proc", version = "0.1.0" }
 openzeppelin-crypto = { path = "lib/crypto" }
 motsu = { path = "lib/motsu" }
-motsu-proc = { path = "lib/motsu-proc", version = "0.1.0" }
+motsu-proc = { path = "lib/motsu-proc", version = ">=0.1, <0.3" }
```

Comment:
Not a huge fan of this. Why don't we just ping it to the latest version? `0.2.0` in this case. What's the benefit of doing `version = ">=0.1, <0.3"`?

Reply:
Answered on Slack.